test/Resiliency.Tests/BackoffStrategies/BackoffStrategyTests.cs

```diff
@@ -24,40 +24,15 @@ public void NegativeInitialValueThrowsException(Type strategyType)
                 {
                     throw tiex.InnerException;
                 }
-                catch (MissingMethodException mex)
+                catch (MissingMethodException)
                 {
                     throw new Exception("The arguments passed into the Activator do not match the constructor of the type being instantiated and may need to be updated.");
                 }
             });
         }
 
         [Theory]
-        [InlineData(typeof(LinearBackoffStrategy))]
-        [InlineData(typeof(ExponentialBackoffStrategy))]
-        public void NegativeAttemptThrowsException(Type strategyType)
-        {
-            TimeSpan initialWaitTime = TimeSpan.FromSeconds(30);
-            var attempt = -1;
-
-            IBackoffStrategy strategy = (IBackoffStrategy)Activator.CreateInstance(strategyType, new object[]{ initialWaitTime });
-            Assert.Throws<ArgumentException>(() => strategy.GetWaitTime(attempt));
-        }
-
-        [Theory]
-        [InlineData(typeof(LinearBackoffStrategy))]
-        [InlineData(typeof(ExponentialBackoffStrategy))]
-        public void ZeroAttemptThrowsException(Type strategyType)
-        {
-            TimeSpan initialWaitTime = TimeSpan.FromSeconds(30);
-            var attempt = 0;
-
-            IBackoffStrategy strategy = (IBackoffStrategy)Activator.CreateInstance(strategyType, new object[]{ initialWaitTime });
-            Assert.Throws<ArgumentException>(() => strategy.GetWaitTime(attempt));
-        }
-
-        [Theory]
-        [InlineData(typeof(HalfJitterBackoffStrategy))]
-        [InlineData(typeof(FullJitterBackoffStrategy))]
+        [InlineData(typeof(MultiplicativeJitterBackoffStrategy))]
```

Comment:
Note to self: Add capped and adjusted decorators into this theory.

Reply:
Constructors are all different enough. I moved this test for each one into their own test classes.